### DISCUSSION TITLE
How to open a new window in react-native-windows?

### DISCUSSION THREAD

**Discussion by kalrit718** [microsoft/react-native-windows#9675] [Q&A] [answered]
What I'm trying to do is open a new app window by pressing a button. And I want to invoke a method in the main window after closing the pop-up window. What is the best way to achieve these requirements?

**Comment by chrisglein** [comment, marked as answer]
[ApplicationView](https://docs.microsoft.com/en-us/uwp/api/Windows.UI.ViewManagement.ApplicationView?view=winrt-22000) is the UWP API for doing multi window stuff. From that doc there should be samples and such. Note that if you're creating a new window that gets a new thread and you'll need another React Native instance. The two React Native hosts can't share an instance.

[a comment by an automated account is not shown]
